test/e2e_node/services/kubelet.go

```diff
@@ -31,7 +31,7 @@ import (
 
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	utilfeature "k8s.io/apiserver/pkg/util/feature"
-	utilflag "k8s.io/apiserver/pkg/util/flag"
+	utilflag "k8s.io/component-base/cli/flag"
```

Comment:
maybe componentflag or something else?

Reply:
componentflag seems long too me, what about `cliflag`?

Reply:
This will also incur much larger code changes through-out instead of just imports.

Reply:
IMO, using `componentflag` and `componentglobalflag` in all change

Reply:
after talking with luxas I changed them all to cliflag.